day_7/fizzbuzz.js

```diff
@@ -0,0 +1,10 @@
+for (let i = 1; i <= 100; i++)
+
+if (i % 3 == 0 && i % 5 == 0) {
+  console.log("FizzBuzz");
+} else if (i % 3 == 0) {
+  console.log("Fizz");
+} else if (i % 5 == 0) {
+  console.log("Buzz")
+} else
```

Comment:
Really clean here, although you should close out your else statement with braces as well